### PR TITLE
add rudrabarad

### LINKED ISSUE #440: Rudra Barad
image

### PR DESCRIPTION
**Pull request by rudrabarad**
resolve #440

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you :smile:

**Comment by AllanRegush** [comment]
Thanks for reviewing @schmelto

**Comment by AllanRegush** [comment]
@rudrabarad Looks like there is a merge conflict. Could you resolve this? If you need any help let us know :nerd_face:

**Comment by rudrabarad** [comment]
> @rudrabarad Looks like there is a merge conflict. Could you resolve this? If you need any help let us know

I've successfully resolved the conflict @AllanRegush  ✅

File: data.json [hunk 1]
```diff
@@ -761,10 +761,18 @@
     "imageUrl": "https://user-images.githubusercontent.com/66346161/114270825-18c90d80-9a2c-11eb-9173-1c072e90ac27.png",
     "issueNumber": 436
   },
+  {
+
```
Comment (schmelto) [inline data.json:765]:
```suggestion
```

File: data.json [hunk 2]
```diff
@@ -761,10 +761,18 @@
...
+    "githubUsername": "rudrabarad",
+    "imageUrl": "https://user-images.githubusercontent.com/53337681/115293769-5983fd80-a175-11eb-91f8-dbdadf06a8b0.png",
+    "issueNumber": 440
+  },
   {
     "name": "Rishabh Rathore",
     "githubUsername": "Rishabhrathore055",
     "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jpg",
     "issueNumber": 445
+
```
Comment (schmelto) [inline data.json:776]:
```suggestion
```

**Review by schmelto — CHANGES_REQUESTED** [review]
maybe a small improvement

**Comment by rudrabarad** [comment]
Can you check now? @AllanRegush 😅

**Review by schmelto — APPROVED** [review]

**Pull request merged**